jenkins/job-configs/kubernetes-jenkins/kubernetes-upgrade.yaml

```diff
@@ -0,0 +1,296 @@
+# This is the old job definition for upgrade tests.
+- job-group:
+    name: 'kubernetes-e2e-{provider}-{version-old}-{version-new}'
+    trigger-job: 'kubernetes-build'
+    test-owner: 'ihmccreery'
```

Comment:
I know I'm going to regret this, but added myself as an owner for GKE upgrade tests.